src/main/java/pl/gispartner/ResourceManagerApp/model/ResourceDto.java

```diff
@@ -0,0 +1,12 @@
+package pl.gispartner.ResourceManagerApp.model;
+
+import lombok.*;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ResourceDto {
+
+    private Long resourceId;
```

Comment:
zostaw po prostu id, jest w obiekcie resourceDto wiec to samo przez sie mowi ze to jest id resourca